mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -50,6 +67,129 @@ def test_brainvision_data_filters():
     assert_equal(raw.info['highpass'], 0.1)
     assert_equal(raw.info['lowpass'], 250.)
 
+    # Heterogeneous highpass in seconds (default measurement unit)
+    with warnings.catch_warnings(record=True) as w:  # event parsing
+        raw = _test_raw_reader(
+            read_raw_brainvision, vhdr_fname=vhdr_mixed_highpass_path,
+            montage=montage, eog=eog)
+
+    trigger_warning = ['parse triggers that' in str(ww.message) for ww in w]
+    lowpass_warning = ['different lowpass filters' in str(ww.message) for ww in w]
+    highpass_warning = ['different highpass filters' in str(ww.message) for ww in w]
+
+    expected_warnings = zip(trigger_warning, lowpass_warning, highpass_warning)
+
+    assert_true(all( any([trg, lp, hp]) for trg, lp, hp in expected_warnings))
```

Comment:
Not sure if I correctly understand this, but this (the 'any') seems a bit vague ... Wouldn't it be more straight-forward AND safer to check the exact counts of the 3 warning types in w? ie., something like

```
w_types = 'parse triggers that', 'different lowpass filters', 'different lowpass filters'
w_num = [1, 2, 3]
for w_type, w_num in zip(w_types, w_num):
    assert_equal(sum([w_type in str(ww.message) for ww in w]), w_num)
```

?